gradle/verification-metadata.xml

```diff
@@ -4122,10 +3589,5 @@
             <sha256 value="282ae185fc2ff27da7714af9962897c09cfefafb88072219c4a2f9c73616c026" origin="Generated by Gradle"/>
          </artifact>
       </component>
-      <component group="zulu_8" name="macos" version="8u302">
```

Comment:
This is also needed for some testing using very old versions of ES that require Java 8.

Reply:
Do we still test on `macos`?

Reply:
Added back via c51a4326e6ea06b9b1e68d0af9f7d865ffc93253